config/azuread.spc

```diff
@@ -1,7 +1,7 @@
 connection "azuread" {
   plugin = "azuread"
 
-  # Defaults to "AZUREPUBLICCLOUD". Valid environments are "AZUREPUBLICCLOUD", "AZURECHINACLOUD", "AZUREGERMANCLOUD" and "AZUREUSGOVERNMENTCLOUD"
+  # Defaults to "AZUREPUBLICCLOUD". Valid environments are "AZUREPUBLICCLOUD", "AZURECHINACLOUD" and "AZUREUSGOVERNMENTCLOUD"
```

Comment:
Do we know if the German cloud works with the older SDK? Was support for the German Cloud changed from Azure in general in the past few months? If not, do we know if the new SDK will support it in the future?

Reply:
From our discussions - German Cloud support has been removed in the Hamilton SDK as well recently, so we will remove it as part of the migration, but can re-add if the MS Graph SDK supports it in the future.